### PR TITLE
Update PBSC cities

### PR DESCRIPTION
**Pull request by madupras**
Added PBSC cities Barcelona, Valence, Buenos Aires, Santiago, Riviera and Monaco

### CODE REVIEW

File: systems.csv [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Country Code,Name,Location,System ID,URL,Auto-Discovery URL
 AE,ADCB Bikeshare,"Abu Dhabi, AE",ABU,https://www.bikeshare.ae/,https://api-core.bikeshare.ae/gbfs/gbfs.json
 AR,MiBiciTuBici,"Rosario, Santa Fe, AR",biketobike,https://www.mibicitubici.gob.ar/,https://www.mibicitubici.gob.ar/opendata/gbfs.json
+AR,Ecobici,"Buenos Aires, AR",bike_buenosaires,https://www.buenosaires.gob.ar/ecobici,https://buenosaires.publicbikesystem.net/ube/gbfs/v1/gbfs.json
```
Comment (jcn) [inline systems.csv:4]: The feeds linked from this auto-discovery are returning "access denied" messages.

File: systems.csv [hunk 2]
```diff
@@ -53,6 +56,7 @@ DE,wupsiRad Leverkusen,"Leverkusen, DE",nextbike_dw,https://www.wupsirad.de/xx/l
 EE,SIXT jalgrattarent,"Tallinn, EE",nextbike_ee,http://www.sixtbicycle.ee/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_ee/gbfs.json
 ES,Bilbaobizi (Bilbao),"Bilbao, ES",nextbike_bo,https://www.bilbaobizi.bilbao.eus/xx/bilbao/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_bo/gbfs.json
 ES,Sitycleta (Las Palmas),"Las Palmas de Gran Canaria, ES",nextbike_el,https://www.sitycleta.com/xx/laspalmas/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_el/gbfs.json
+FR,Libélo, "Valence, FR",libelo,http://www.citea.info/presentation/?rub_code=92,https://valence.publicbikesystem.net/ube/gbfs/v1/gbfs.json
```
Comment (jcn) [inline systems.csv:60]: The feeds in this auto-discovery file are returning blank.

**Review by jcn — CHANGES_REQUESTED** [review]
Thanks for the contribution! Unfortunately, two of the feeds are not returning valid information and should be removed before we can merge in this PR. Thanks.

File: systems.csv (hunk 1 above)
Comment (madupras) [reply systems.csv:4]: Buenos Aires auto-discovery links are fixed

File: systems.csv (hunk 2 above)
Comment (madupras) [reply systems.csv:60]: A system update is needed to fix the Valence links. It will be upgraded in the coming weeks

File: systems.csv [hunk 3]
```diff
@@ -131,6 +136,7 @@ SC, Just Eat Cycles,"Edinburgh, SC",edinburgh-city-bikes,https://edinburghcycleh
 SI,NomagoBikes (Slovenia),"SI",nextbike_cn,https://bikes.nomago.si/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_cn/gbfs.json
 SK,Arriva Nitra Slovakia,"Nitra, SK",nextbike_as,https://arriva.bike/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_as/gbfs.json
 SK,BikeKIA,"Žilina, SK",nextbike_ak,https://bikekia.sk/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_ak/gbfs.json
+SP,Bicing,"Barcelona, SP",bike_barcelona,https://www.bicing.barcelona/,https://barcelona.publicbikesystem.net/ube/gbfs/v1/gbfs.json
```
Comment (bddq) [inline systems.csv]: Spain country code is `ES` not `SP`.

File: systems.csv [hunk 4]
```diff
@@ -16,13 +17,15 @@ BR,Bike Itaú - Poa,"Porto Alegre, BR",bike_poa,https://bikeitau.com.br/bikepoa,
 BR,Bike Itaú - Rio,"Rio de Janeiro, BR",bike_rio,https://bikeitau.com.br/bikerio/,https://riodejaneiro.publicbikesystem.net/ube/gbfs/v1/
 BR,Bike Itaú - Salvador,"Salvador, BR",bike_salvador,https://bikeitau.com.br/bikesalvador/,https://salvador.publicbikesystem.net/ube/gbfs/v1/
 BR,Bike Itaú - Sampa,"São Paulo, BR",bike_sampa,https://bikeitau.com.br/bikesampa,https://saopaulo.publicbikesystem.net/ube/gbfs/v1/
+BR,Bike Itaú - Riviera,"Riviera, BR",https://www.rivieradesaolourenco.com/rivibike-2/,https://riviera.publicbikesystem.net/ube/gbfs/v1/
```
Comment (bddq) [inline systems.csv]: System ID is missing here.

**Review by bddq — CHANGES_REQUESTED** [review]

**Review by bddq — APPROVED** [review]

File: systems.csv (hunk 2 above)
Comment (madupras) [reply systems.csv:60]: The Valence system has been updated and the feed is now correct

Comment (madupras) [reply systems.csv:60]: https://valence.publicbikesystem.net/ube/gbfs/v1/

**Review by jcn — APPROVED** [review]

**Pull request merged**